modules/ConnectedFetch.js

```diff
@@ -15,13 +15,15 @@ const createConnectedFetch = (): Class<*> => {
       ? this.context.store.getState()
       : this.props.store
     rdfTimeout: ?number = this.props.timeout
+    rdfOnIntercept: ?OnInterceptFn = this.props.onIntercept
```

Comment:
Therefore we'll have here : `rdfInterceptor: ?Interceptor = this.props.onIntercept`